core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -93,20 +93,20 @@ var SkillEditorPage = function() {
     by.css('.protractor-test-question-list-item'));
   var questionItem = element(by.css('.protractor-test-question-list-item'));
 
-  var editRubricExplanationButtons = element.all(
-    by.css('.protractor-test-edit-rubric-explanation'));
+  var addRubricExplanationButton = element.all(
+    by.css('.protractor-test-add-explanation-button'));
   var saveRubricExplanationButton = element(
     by.css('.protractor-test-save-rubric-explanation-button'));
-  var rubricExplanations = element.all(
-    by.css('.protractor-test-rubric-explanation'));
+  var deleteRubricExplanationButton = element(
+    by.css('.protractor-test-delete-rubric-explanation-button'));
 
   this.get = function(skillId) {
     browser.get(EDITOR_URL_PREFIX + skillId);
     return waitFor.pageToFullyLoad();
   };
 
-  this.editRubricExplanationWithIndex = function(index, explanation) {
-    editRubricExplanationButtons.then(function(buttons) {
+  this.addRubricExplanationToDifficultyWithIndex = function(index, explanation) {
```

Comment:
Can we use the name of the difficulty instead of its index?

Reply:
Done